<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_opfiltro_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1881</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 2px solid #dcdfe6;
border-radius: 15px;
gridline-color: #e0e0e0;
outline: none;
color: #606266;
border-radius: 15px;
border-bottom: 1px solid #E3E3E3;

</string>
    </property>
    <widget class="QLabel" name="corridas_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>551</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>59</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="header_corridas" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1951</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-bottom: 1px solid #E3E3E3;</string>
    </property>
    <widget class="QPushButton" name="boton_rutas">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size:25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string>Rutas</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/entrega.png</normaloff>../recursos/recursos empresa/entrega.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_corridas">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string>Corridas</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/calendario.png</normaloff>../recursos/recursos empresa/calendario.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_operadores">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string>Operadores</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/conductor.png</normaloff>../recursos/recursos empresa/conductor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_autobuses">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string>Autobuses</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/autobus.png</normaloff>../recursos/recursos empresa/autobus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_inicio">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/casa-silueta-negra-sin-puerta.png</normaloff>../recursos/recursos empresa/casa-silueta-negra-sin-puerta.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_reservaciones">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string>Reservaciones</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/boleto.png</normaloff>../recursos/recursos empresa/boleto.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_salir">
     <property name="geometry">
      <rect>
       <x>1640</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/recursos empresa/cerrar-sesion.png</normaloff>../recursos/recursos empresa/cerrar-sesion.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="boton_autobuses_2">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>960</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 25px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
    </property>
    <property name="text">
     <string>Comprar boleto</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../recursos/recursos empresa/boleto.png</normaloff>../recursos/recursos empresa/boleto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>951</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>41</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pasajeros</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="pasajerosm_tableWidget">
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 15px;
    gridline-color: #e0e0e0;
    outline: none;
    font-size: 13px;
    color: #606266;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #409eff;
    color: white;
}

QTableWidget::item:hover {
    background-color: #f5f7fa;
}

/* Header con bordes redondeados solo arriba */
QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}

QHeaderView::section {
    background-color: #1061C4;
    color: white;
    font-weight: bold;
    font-size: 18px;
    padding: 8px 12px;
    border: none;
    border-right: 1px solid #0a4a9d;
}

QHeaderView::section:hover {
    background-color: #0a4a9d;
}

QHeaderView::section:pressed {
    background-color: #083a7a;
}
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>237</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>N° Pasajero</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Edad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Telefono</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>260</y>
      <width>751</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>41</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Corridas activas</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
border:none;</string>
         </property>
         <property name="text">
          <string>Buscar por fecha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_filtroFecha">
         <property name="maximumDateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2030</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2025</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="maximumDate">
          <date>
           <year>2030</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>2025</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="corridasm_tableWidget">
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 15px;
    gridline-color: #e0e0e0;
    outline: none;
    font-size: 15px;
    color: #606266;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #409eff;
    color: white;
}

QTableWidget::item:hover {
    background-color: #f5f7fa;
}

/* Header con bordes redondeados solo arriba */
QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}

QHeaderView::section {
    background-color: #1061C4;
    color: white;
    font-weight: bold;
    font-size: 18px;
    padding: 8px 12px;
    border: none;
    border-right: 1px solid #0a4a9d;
}

QHeaderView::section:hover {
    background-color: #0a4a9d;
}

QHeaderView::section:pressed {
    background-color: #083a7a;
}
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>187</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>N° Corrida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Hora Salida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ruta</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>41</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Operadores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="autobusm_tableWidget">
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 15px;
    gridline-color: #e0e0e0;
    outline: none;
    font-size: 13px;
    color: #606266;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #409eff;
    color: white;
}

QTableWidget::item:hover {
    background-color: #f5f7fa;
}

/* Header con bordes redondeados solo arriba */
QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}

QHeaderView::section {
    background-color: #1061C4;
    color: white;
    font-weight: bold;
    font-size: 18px;
    padding: 8px 12px;
    border: none;
    border-right: 1px solid #0a4a9d;
}

QHeaderView::section:hover {
    background-color: #0a4a9d;
}

QHeaderView::section:pressed {
    background-color: #083a7a;
}
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>187</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>N° Operador</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Corrida asignada</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha de corrida</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
